experimental/dds/tree2/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -30,31 +31,31 @@ import { Summarizable, SummaryElementParser, SummaryElementStringifier } from ".
 import { isJsonObject, JsonCompatible, JsonCompatibleReadOnly } from "../util";
 import { makeSchemaCodec, Format, encodeRepo } from "./schemaIndexFormat";
 
-/**
- * The storage key for the blob in the summary containing schema data
- */
-const schemaBlobKey = "SchemaBlob";
-
 const schemaStringKey = "SchemaString";
 
+interface CollabWindow {
```

Comment:
see other doc comment. this should be exported from the file so that it can be referenced by someone instantiating `SchemaSummarizer` if they want.

Reply:
also, needs top-level doc comment

Reply:
updated `CollabWindow` documentation